src/screens/SavingsSheet.js

```diff
@@ -19,24 +19,23 @@ import {
 } from '../components/sheet';
 import Divider from '../components/Divider';
 import { convertAmountToDepositDisplay } from '../helpers/utilities';
+import { DAI_ADDRESS } from '../references';
 
 const SavingsSheet = () => {
-  console.log('[SAVINGS SHEET]');
   const { getParam, navigate } = useNavigation();
 
   const isEmpty = getParam('isEmpty');
+  const nativeValue = getParam('nativeValue');
+  const isStablecoin = getParam('isStablecoin');
   const underlying = getParam('underlying');
   const lifetimeSupplyInterestAccrued = getParam(
     'lifetimeSupplyInterestAccrued'
   );
   const supplyBalanceUnderlying = getParam('supplyBalanceUnderlying');
   const supplyRate = getParam('supplyRate');
 
-  // TODO JIN transactions list
-  const balance = convertAmountToDepositDisplay(supplyBalanceUnderlying, {
-    address: underlying.address,
-    decimals: underlying.decimals,
-    symbol: underlying.symbol,
+  const balance = convertAmountToDepositDisplay(nativeValue, {
+    address: DAI_ADDRESS,
```

Comment:
@jinchung  I just want to render `$ XX.YY` so if there's a better way of doing this please LMK!